### DISCUSSION TITLE
Cluster QE Issue tracker

### DISCUSSION THREAD

**Discussion by cmackows** [teemtee/tmt#1137] [General]
List of issues blocking Cluster-QE use of TMT with libvirt host.

Phase 1.  # Simple HA cluster with proper fencing configuration

	1. Name Resolution 
	Currently with TMT multihost, the guests do not have name resolution. Per Petr, upcoming release will populate /etc/hosts. 

	2. VM machine name #1136 
	The OS hostname is not same as kvm machine name. Right now we do not have exposure to kvm machine name for fencing. 
	https://github.com/teemtee/tmt/issues/1136

Phase 2.  # Full HA Cluster with shared storage and shared IP resources

	1. Shared storage #1133 
	GFS2/LVM2 tests will require shared storage for cluster members        
	https://github.com/teemtee/tmt/issues/1133

	2. Shared IP  #1134 
	Floating or Virtual IP provided but not assigned to any host
	https://github.com/teemtee/tmt/issues/1134

**Comment by psss** [comment]
> Per Petr, upcoming release will populate /etc/hosts

The latest `tmt-1.12` already includes a simple guest setup including `/etc/hosts` adjustment. Simple example test for experimenting available in #1135.

**Reply by cmackows** [reply]
@psss  I have verified /etc/hosts fix using tmt-1.12.0-1.20220426132939307433

[root@testcloud tree]# cat /etc/hosts
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
192.168.122.140 server
192.168.122.238 client-one
192.168.122.14 client-two

**Reply by psss** [reply]
Nice! Thanks for the verification.
